src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,15 @@
+import './Homepage.css';
+import { Navigation } from "../Navigation/Navigation";
+import { SideBar } from './Sidebar';
+import { MainContent } from './MainContent';
+
+ export const HomePage = () => {
+
+    return <div className='homePage'>
+        <Navigation />
```

Comment:
add the navigation to the routes component that way you don't have to import the navigation component on every page